lib/rubocop/config_loader.rb

```diff
@@ -81,25 +84,29 @@ def merge(base_hash, derived_hash)
         result
       end
 
-      def base_configs(path, inherit_from)
+      def base_configs(path, inherit_from, file)
         configs = Array(inherit_from).compact.map do |f|
-          if f =~ /\A#{URI::Parser.new.make_regexp(%w[http https])}\z/
-            f = RemoteConfig.new(f, File.dirname(path)).file
-          else
-            f = File.expand_path(f, File.dirname(path))
-
-            if auto_gen_config?
-              next if f.include?(AUTO_GENERATED_FILE)
-            end
-
-            print 'Inheriting ' if debug?
-          end
-          load_file(f)
+          f = load_inherited(path, f, file)
+          load_file(f) unless f.nil?
         end
 
         configs.compact
       end
 
+      def load_inherited(path, inherit_from, file)
```

Comment:
It's a bit unclear what this method does, and whether it does a single thing. Naming it as an imperative verb suggests it is used for a side effect, but it seems the return value is what's relevant, so it could probably be better named by what it returns. 🙂